linera-chain/src/manager.rs

```diff
@@ -196,15 +198,16 @@ impl ChainManager {
             distribution,
             fallback_distribution,
             proposed: None,
+            proposed_pending_blobs: BTreeMap::new(),
             locked: None,
+            locked_pending_blobs: BTreeMap::new(),
```

Comment:
Since there is a field called `pending`, maybe better names will be `proposed_blobs` and `locked_blobs` ?

Reply:
Or `blobs_proposed`, `blobs_locked`.